Pod/Classes/Image Categories/FLAnimatedImageView+PINRemoteImage.m

```diff
@@ -10,92 +10,92 @@
 }
 
-- (void)setImageFromURL:(NSURL *)url processorKey:(NSString *)processorKey processor:(PINRemoteImageManagerImageProcessor)processor completion:(PINRemoteImageManagerImageCompletion)completion
+- (void)pin_setImageFromURL:(NSURL *)url processorKey:(NSString *)processorKey processor:(PINRemoteImageManagerImageProcessor)processor completion:(PINRemoteImageManagerImageCompletion)completion
 {
     [PINRemoteImageCategoryManager setImageOnView:self fromURL:url processorKey:processorKey processor:processor completion:completion];
 }
 
-- (void)setImageFromURL:(NSURL *)url placeholderImage:(UIImage *)placeholderImage processorKey:(NSString *)processorKey processor:(PINRemoteImageManagerImageProcessor)processor completion:(PINRemoteImageManagerImageCompletion)completion
+- (void)pin_setImageFromURL:(NSURL *)url placeholderImage:(UIImage *)placeholderImage processorKey:(NSString *)processorKey processor:(PINRemoteImageManagerImageProcessor)processor completion:(PINRemoteImageManagerImageCompletion)completion
 {
     [PINRemoteImageCategoryManager setImageOnView:self fromURLs:@[url] placeholderImage:placeholderImage processorKey:processorKey processor:processor completion:completion];
 }
 
-- (void)setImageFromURLs:(NSArray *)urls
+- (void)pin_setImageFromURLs:(NSArray *)urls
 {
     [PINRemoteImageCategoryManager setImageOnView:self fromURLs:urls];
 }
 
-- (void)setImageFromURLs:(NSArray *)urls placeholderImage:(UIImage *)placeholderImage
+- (void)pin_setImageFromURLs:(NSArray *)urls placeholderImage:(UIImage *)placeholderImage
 {
     [PINRemoteImageCategoryManager setImageOnView:self fromURLs:urls placeholderImage:placeholderImage];
 }
 
-- (void)setImageFromURLs:(NSArray *)urls placeholderImage:(UIImage *)placeholderImage completion:(PINRemoteImageManagerImageCompletion)completion
+- (void)pin_setImageFromURLs:(NSArray *)urls placeholderImage:(UIImage *)placeholderImage completion:(PINRemoteImageManagerImageCompletion)completion
 {
     [PINRemoteImageCategoryManager setImageOnView:self fromURLs:urls placeholderImage:placeholderImage completion:completion];
 }
 
-- (void)cancelImageDownload
+- (void)pin_cancelImageDownload
 {
     [PINRemoteImageCategoryManager cancelImageDownloadOnView:self];
 }
 
-- (NSUUID *)downloadImageOperationUUID
+- (NSUUID *)pin_downloadImageOperationUUID
 {
     return [PINRemoteImageCategoryManager downloadImageOperationUUIDOnView:self];
 }
 
-- (void)setDownloadImageOperationUUID:(NSUUID *)downloadImageOperationUUID
+- (void)pin_setDownloadImageOperationUUID:(NSUUID *)downloadImageOperationUUID
 {
     [PINRemoteImageCategoryManager setDownloadImageOperationUUID:downloadImageOperationUUID onView:self];
 }
 
-- (BOOL)updateWithProgress
+- (BOOL)pin_updateWithProgress
 {
     return [PINRemoteImageCategoryManager updateWithProgressOnView:self];
 }
 
-- (void)setUpdateWithProgress:(BOOL)updateWithProgress
+- (void)setPin_updateWithProgress:(BOOL)updateWithProgress
```

Comment:
That's what I was wondering, but then I looked at the others and they didn't match the pattern. Now I see this is the only one declared as a property. I swear I wrote this thing ;)